tests/models/test_user.py

```diff
@@ -36,4 +36,11 @@ def test_create_user(db, cleanup):
     assert user.username == username
     assert user.image == image
 
+def test_user_image_has_default_value(db, cleanup):
+    email = "dan@example.com"
+    password = "123456"
+    username = "chunky_lover"
+    user_in = UserCreate(email=email, password=password, username=username)
+    user = crud.create_user(db, user=user_in)
+    assert user.image == ''
```

Comment:
Test is passing!

Reply:
Great work Zack, thank you. Now the user profile can get rolling lol